redis/commands.cc

```diff
@@ -25,13 +25,16 @@
 #include "redis/reply.hh"
 #include "types.hh"
 #include "service_permit.hh"
-#include "service/storage_proxy.hh"
 #include "service/client_state.hh"
 #include "redis/options.hh"
 #include "redis/query_utils.hh"
 #include "redis/mutation_utils.hh"
 #include "redis/lolwut.hh"
 
+namespace service {
+    class storage_proxy;
```

Comment:
Style: We don't indent inside namespace block.

Reply:
Now the forward declaration is somewhere else, this time uses the proper style.